cg/store/api/status.py

```diff
@@ -208,6 +216,19 @@ def cases(
 
         return sorted(cases, key=lambda k: k["tat"], reverse=True)
 
+    def set_case_action(self, action: Optional[str], case: Union[str, List[str]]):
```

Comment:
```suggestion
    def set_case_action(self, action: Literal[CASE_ACTIONS], case_id: str)-> None:
```
Simplicity is better! Having a choice leads to more complexity and unpredictable behaviors later